ironfish/src/wallet/wallet.ts

```diff
@@ -814,6 +814,7 @@ export class Wallet {
     expiration?: number
     expirationDelta?: number
     confirmations?: number
+    fund?: boolean
```

Comment:
do we need the optional funding? if there aren't any outputs or burns i don't think there's anything to fund.